docs/getting-started-guides/mesos/index.md

```diff
@@ -313,25 +313,23 @@ For support level information on all solutions, see the [Table of solutions](/do
 
 ## What next?
 
-Try out some of the standard [Kubernetes examples][9].
+Try out some of the standard [Kubernetes examples](https://releases.k8s.io/{{page.githubbranch}}/examples/).
```

Comment:
You can just make the URL use "master" since it's unlikely to change, i.e. `https://releases.k8s.io/master/examples/`